lib/rules/space-infix-ops.js

```diff
@@ -51,7 +53,7 @@ module.exports = {
 
         // Build maps of operators that must or must not
         // have spaces around them.
-        const spacesYes = {};
+        const spacesYes = {in: true, instanceof: true};
```

Comment:
I added space inside the braces.
I understand this needs to be rebased. Unfortunately rebasing has never gone well for me. Can you tell me the steps to successfully do that? The last few times I tried rebasing (as opposed to merging) the code got messed up really bad.